.github/workflows/check.yml

```diff
@@ -0,0 +1,68 @@
+name: Flatpak CI
+
+on: [push]
+
+jobs:
+  check:
+    name: Build with gcc and test
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Install Dependencies
+      run: >
```

Comment:
Just a yaml thing but I think this is cleaner:

```yaml
run: |
    sudo apt-add-repository foo
    sudo apt-get update
    sudo apt-get install -y
```

Reply:
Then that makes the actual install line super wide though... With this i can linebreak the list of dependencies however i want.